_posts/en/newsletters/2023-05-17-newsletter.md

```diff
@@ -0,0 +1,242 @@
+---
+title: 'Bitcoin Optech Newsletter #251'
+permalink: /en/newsletters/2023/05/17/
+name: 2023-05-17-newsletter
+slug: 2023-05-17-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal to begin testing HTLC
+endorsement, relays a request for feedback about proposed specifications
+for Lightning Service Providers (LSPs), discusses challenges with
+opening zero-conf channels when using dual funding, looks at a
+suggestion for advanced applications of payjoin transactions, and links
+to summaries of a recent in-person meeting of Bitcoin Core developers.
+Also included in this week's newsletter is the first part of a new
+series about policies for transaction relay and mempool inclusion, plus
+our regular sections announcing new releases and release candidates and
+describing notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Testing HTLC endorsement:** several weeks ago, Carla Kirk-Cohen and
+  Clara Shikhelman [posted][kcs endorsement] to the Lightning-Dev mailing list about
+  the next steps they and others planned to take to test the idea of
+  [HTLC][topic htlc] endorsement (see [Newsletter #239][news239
+  endorsement]) as part of a mitigation for [channel jamming
+  attacks][topic channel jamming attacks].  Most notably, they provided a
+  short [proposed specification][bolts #1071] that could be deployed
+  using an experimental flag, preventing deployments of it from having any effect on
+  interactions with non-participating nodes.
```

Comment:
Today, an email [0] was added to the thread, listing 4 common misconceptions about the reputation scheme. The email reiterates that:

1.  There is no gossip in the scheme
2. Less active edges will experience a gradually degraded quality of service only during a jamming attack
3. Reputation is difficult to gain, needs to be continuously maintained, and is lost easily
4. Reputation gained during times of low activity cannot be used in times of high activity


[0] https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-May/003958.html